frontend/src/modules/tournaments-list/pages/TournamentsListPage,js.js

```diff
@@ -1,18 +1,29 @@
 import { ContentBox } from 'src/shared/design-system';
-import { RouterLink } from 'src/shared/navigation';
+import { Box } from '@chakra-ui/react';
+import CreateNewButton from '../atoms/CreateNewButton';
+import TournamentTable from '../organisms/TournamentTable';
+import TournamentListHeading from '../atoms/TournamentListHeading';
+import { useState } from 'react';
+import TournamentFilter from '../organisms/TournamentFilter';
 
 const TournamentsListPage = () => {
```

Comment:
Ještě by to chtělo přidat TournamentListTemplate, page má jenom přidávat data do toho template. Můžeš se kouknout na to, co myslim, třeba v LoginPage